src/main/java/io/supertokens/useridmapping/UserIdMapping.java

```diff
@@ -44,11 +46,45 @@
 
 public class UserIdMapping {
 
-    public static void createUserIdMapping(AppIdentifierWithStorage appIdentifierWithStorage,
+    public static void createUserIdMapping(Main main, AppIdentifierWithStorage appIdentifierWithStorage,
                                            String superTokensUserId, String externalUserId,
                                            String externalUserIdInfo, boolean force)
             throws UnknownSuperTokensUserIdException,
-            UserIdMappingAlreadyExistsException, StorageQueryException, ServletException, TenantOrAppNotFoundException {
+            UserIdMappingAlreadyExistsException, StorageQueryException, ServletException,
+            TenantOrAppNotFoundException {
+
+        // Check if mapping already exists app wide
```

Comment:
add more comments as to why this is being done

Reply:
Done